src/huggingface_hub/hf_api.py

```diff
@@ -222,7 +222,7 @@ def __init__(
         self.modelId = modelId
         self.sha = sha
         self.lastModified = lastModified
-        self.tags = tags
+        self.tags = tags if tags is not None else []
```

Comment:
```suggestion
        self.tags = tags
```
I'd even do this personally (i.e. self.tags can be None)